docs/stacks-guide-azure.md

```diff
@@ -0,0 +1,429 @@
+6. [Configure Provider Resources](#configure-provider-resources)
+7. [Recap](#recap)
+8. [Next Steps](#next-steps)
+
+## Introduction
+
+In this guide, we will set up a Azure provider in Crossplane so that we can
+install and use the WordPress sample stack, which depends on MySQL and
+Kubernetes!
+
+Before you begin, you will need:
+
+* Everything from the [Crossplane Stacks Guide][stacks-guide] before the cloud
+  provider setup
+  - A `kubectl` pointing to a Crossplane control cluster
+  - The [Crossplane CLI][crossplane-cli] installed
+* An account on [Azure][azure]
+
+At the end, we will have:
+
+* A Crossplane control cluster configured to use Azure
+* The boilerplate of a Azure-based project spun up
+* Support in the control cluster for managing MySQL and Kubernetes cluster
+  dependencies
+* A slightly better understanding of:
+  - The way cloud providers are configured in Crossplane
+  - The way dependencies for cloud-portable workloads are configured in
+    Crossplane
+
+We will **not** be teaching first principles in depth. Check out the [concepts
+document][crossplane-concepts] for that.
+
+## Install the Azure Stack
+
+After Crossplane has been installed, it can be extended with more functionality
+by installing a Crossplane Stack! Let's install the stack for Microsoft Azure to
+add support for that cloud provider. We can use the Crossplane CLI for this
+operation:
+
+```
+kubectl crossplane stack install 'crossplane/stack-azure:master' stack-azure
+```
+To install to a particular namespace, you can use the `generate-install` command
+and pipe it to `kubectl apply` instead, which gives you more control over how
+the stack's installation is handled. Everything is Kubernetes object!
+
+Since this is an infrastructure stack, we need to specify that it's
+cluster-scoped via `--cluster` flag.
+```
+kubectl crossplane stack generate-install --cluster 'crossplane/stack-azure:master' stack-azure | kubectl apply --namespace crossplane-system -f -
+```
+
+We have installed the Azure stack into the `crossplane-system` namespace, but we
+want to group our Azure-specific resources in their own environment namespaces.
+For the purpose of this guide, we will create all Azure-specific resources in
+the `azure-infra-dev` namespace, and all application-specific resources in the
+`app-project1-dev` namespace. Let's create these namespaces before we get
+started:
+
+```
+kubectl create namespace azure-infra-dev
+kubectl create namespace app-project1-dev
+```
+
+## Configure Azure Account
+
+We will make use of the following services on Azure:
+
+*   Resource Group
+*   AKS
+*   Azure Database for MySQL
+*   Virtual Network
+*   Subnetwork
+*   Virtual Network Rule
+
+In order to utilize each of these services, you will need to follow the Adding
+Microsoft Azure to Crossplane [guide](cloud-providers/azure/azure-provider.md)
+to obtain appropriate credentials in a JSON file referred to as
+`crossplane-azure-provider-key.json`.
```

Comment:
Hmm, it'd be great if we could include the instructions here so the user doesn't have to open up another document! They seem like they might be able to fit well.

Reply:
This is a somewhat lengthy process that I believe may add clutter this guide if included.